apps/trading/client-pages/markets/use-markets-stats.tsx

```diff
@@ -0,0 +1,96 @@
+import type { MarketMaybeWithCandles } from '@vegaprotocol/markets';
+import { AccountType } from '@vegaprotocol/types';
+import { addDecimal, priceChangePercentage } from '@vegaprotocol/utils';
+import { orderBy } from 'lodash';
+
+/**
+ * useTotalVolume24hCandles returns 24 hr candles with total volume
+ * this is used to draw the sparkline
+ *
+ * @param activeMarkets
+ * @returns
+ */
+export const useTotalVolume24hCandles = (
+  activeMarkets: MarketMaybeWithCandles[] | null
+) => {
+  const candles = [];
+  if (!activeMarkets || activeMarkets.length === 0) return [];
+  for (let i = 0; i < 24; i++) {
+    const totalVolume24hr = activeMarkets.reduce((acc, market) => {
+      const c = market.candles?.[i];
+      if (!c) return acc;
+      return (
+        acc +
+        Number(
+          addDecimal(
+            c.notional,
```

Comment:
We need to convert with quantum first to get something close to the actual usd amount